### PR TITLE
Pull req1  A.java

### PR DESCRIPTION
**Pull request by pr714**
test

### CODE REVIEW

**Comment by PoonamKum** [comment]
change the code

File: A.java [hunk 1]
```diff
@@ -4,5 +4,5 @@
 public static void main(String args[])
 {
 	System.out.println("Hello Git");
-	System.out.println("Some changes");
+
```
Comment (pr714) [inline A.java:7]: why to remove this line

**Review by pr714 — COMMENTED** [review]
changes done

File: A.java (hunk 1 above)
Comment (pr714) [reply A.java:7]: sure

Comment (pr714) [reply A.java:7]: done

**Pull request closed without merging**